lib/gitlab.rb

```diff
@@ -21,11 +21,16 @@ def self.client(options = {})
     Gitlab::Client.new(options)
   end
 
-  # Delegate to Gitlab::Client
-  def self.method_missing(method, *args, &block)
-    return super unless client.respond_to?(method)
-
-    client.send(method, *args, &block)
+  if RUBY_VERSION >= '3.0.0'
+    def self.method_missing(method, *args, **keywargs, &block)
+      return super unless client.respond_to?(method)
```

Comment:
Could you add an empty line after just to fix the rubocop offense?

Reply:
👍🏻